Automattic/wp-calypso
# Stats: remove site setup checklist banner

This completely removes the "banner" ViewMode from the checklist component, and ensures that we don't see the following in stats:

![60628965-70e3d200-9db9-11e9-9133-28a4d0e1adcb](https://user-images.githubusercontent.com/1270189/77374465-051b0e00-6d28-11ea-8c57-4e0661dfd6e7.png)

Fixes https://github.com/Automattic/wp-calypso/issues/38172

### Testing Instructions
- Create a new site, verify that the banner does not appear in /stats
- No regressions / new JS errors from missing usages
